packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -140,7 +140,7 @@ class TransactionService {
         type,
         perfOptions
       )
-      tr.redefine(name, type, perfOptions)
+      tr.redefine(name, undefined, perfOptions)
```

Comment:
can you explain and also add a comment on why we do set the argument to `undefined`.

Reply:
I've added some comments in the code!